cfg/test_info.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "Running test.sh"
```

Comment:
Can we run this by default? Seems more useful than just shutting down

Reply:
That would be my preference, but I was erring on the side of don't touch the defaults, hence the new script. I can make the existing cfg/test.sh look like this, or we can keep both and I can just point to test_info.sh by default. Preference?

Reply:
My preference is to have test_shutdown.sh and test_info.sh and point to test_info by default

Reply:
I will rename test.sh to test_shutdown.sh here. Pointing to test_info by default will happen in the SDK PR.